pkg/controller/remotemachineset/gcpactuator.go

```diff
@@ -163,8 +173,10 @@ func (a *GCPActuator) GenerateMachineSets(cd *hivev1.ClusterDeployment, pool *hi
 	ic := &installertypes.InstallConfig{
 		Platform: installertypes.Platform{
 			GCP: &installertypesgcp.Platform{
-				Region:    cd.Spec.Platform.GCP.Region,
-				ProjectID: a.projectID,
+				Region:        cd.Spec.Platform.GCP.Region,
+				ProjectID:     a.projectID,
+				ComputeSubnet: a.subnet,
+				Network:       a.network,
```

Comment:
also add unit test that checks the network and subnet match when the actuator is configured with those.

Reply:
Will do.